app/containers/ItunesContainer/index.js

```diff
@@ -61,28 +94,107 @@ export function ItunesContainer({
+    setTuneId(trackId);
+    audio?.pause();
+    const currentAudio = new Audio(url);
+    setAudio(currentAudio);
+    if (current && url === trackUrl) {
+      currentAudio.currentTime = current;
+    }
+    setTrackUrl(url);
+    currentAudio.play();
+  };
+
+  const pauseSong = () => {
+    setIsPlaying(false);
+    audio?.pause();
+    setCurrent(audio?.currentTime);
+  };
+  const renderTrack = () => {
+    const items = get(itunesData, 'results', []);
+    const resultCount = get(itunesData, 'resultCount', 0);
+    return (
+      (items.length !== 0 || loading) && (
+        <Skeleton datat-testid="skeleton" loading={loading} active>
+          <If condition={artistName}>
+            <div>
+              <CustomisedT id="search_query" values={{ artistName }} />
+            </div>
+          </If>
+          <If condition={resultCount !== 0}>
+            <div>
+              <CustomisedT id="matching_tracks" values={{ resultCount }} />
+            </div>
+          </If>
+          <For
+            data-testid="music-grid"
+            of={items}
+            ParentComponent={MusicGrid}
+            renderItem={(item, index) => (
+              <TrackCard
+                data-testid="track-card"
+                item={item}
+                key={index}
+                currentPlayingId={tuneId}
+                currentTrack={audio}
+                isPlaying={isPlaying}
+                onPlay={playSong}
+                onPause={pauseSong}
+              />
+            )}
+          />
+        </Skeleton>
+      )
+    );
+  };
+  const renderErrorState = () => {
+    let trackError;
+    if (itunesError) {
+      trackError = itunesError;
+    } else if (!get(itunesData, 'resultCount', 0)) {
+      trackError = 'artist_search_default';
+    }
+    return (
+      !loading &&
+      trackError && (
+        <CustomCard color={itunesError ? 'red' : 'grey'} title={intl.formatMessage({ id: 'Itunes_list' })}>
+          <T id={trackError} />
+        </CustomCard>
+      )
+    );
+  };
+
   return (
-    <Container>
-      <CustomCard>
-        <T marginBottom={10} id="artist_search" />
-        <Search
-          data-testid="search-bar"
-          defaultValue={artistName}
-          type="text"
-          onChange={(evt) => debouncedHandleOnChange(evt.target.value)}
-          onSearch={(searchText) => debouncedHandleOnChange(searchText)}
-        />
-      </CustomCard>
-    </Container>
+    <>
+      <Container padding={padding} maxwidth={maxwidth}>
+        <CustomCard maxwidth={maxwidth}>
+          <T marginBottom={10} id="artist_search" />
+          <Search
+            data-testid="search-bar"
+            defaultValue={artistName}
+            type="text"
+            onChange={(evt) => debouncedHandleOnChange(evt.target.value)}
+            onSearch={(searchText) => debouncedHandleOnChange(searchText)}
+          />
+        </CustomCard>
+      </Container>
+      <BottomContainer maxWidth={trackWidth}>
```

Comment:
this should be `maxwidth` right?

Reply:
no actully i have used different width for this block

Reply:
but the prop that you use in the maxwidth. Could you inspect and see if bottom container is actually reading the value?

Reply:
yes i have checked just now again , its using the value

Reply:
I don't think so. 
![screen](https://user-images.githubusercontent.com/89914586/133444579-06dc2d57-1e5a-418e-bbee-d6c024ae6fb8.png)
See how the max-width is invalid for the bottomcontainer

Reply:
yes you are right ...fixed now